### ISSUE TITLE
uncaught exception: TypeError: db[collInfo.coll].stats is not a function

### ISSUE THREAD

**Issue by josefahmad** [johnlpage/MongoCacheView#8]
```
$ git rev-parse --short HEAD
21f2d9b
```
```
$ mongo mongocacheview.js
Collection                                                           	Size	Cached	%tage	Delta	Read	Written	Used
[some (all?) collections listed]
uncaught exception: TypeError: db[collInfo.coll].stats is not a function :
@mongocacheview.js:50:21
failed to load: mongocacheview.js
exiting with code -3
$
```

**Comment by josefahmad** [comment]
Figured out it occurs when there's a collection named `version`, which clashes with the `db.version()` shell helper. The fix should be to use the `db.getCollection()` helper instead.

**Comment by josefahmad** [comment]
Verified as fixed in commit fba18017e1aa7668dcc8bb7f162acc8afa473b18.
